recipes/sundials/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cc -I${PREFIX}/include -L${PREFIX}/lib -o cvRoberts_dns cvRoberts_dns.c -lm -lsundials_cvode -lopenblas -lsundials_nvecserial
```

Comment:
I do not understand `SUNDIALS` but if this test is enough just ignore my comments above.